pipeline_tools/tests/test_get_upload_urn.py

```diff
@@ -21,21 +21,21 @@ def setUp(self):
 
     def test_get_staging_urn_empty_js(self):
         js = {}
-        self.assertIsNone(gsu.get_staging_urn(js))
+        self.assertIsNone(getter.get_staging_urn(js))
```

Comment:
nit: update the function names and calls here if we want to change the func name to `get_upload_urn`